Split swdd requirements for agent resource monitoring (#571)
eclipse-ankaios/ankaios: agent/doc/swdesign/README.md

Status: approved

At an interval of 2 seconds, the AgentManager shall:

> * request the ResourceMonitor to provide the resource metrics for the agent's node

Reviewer: Just a small clarification as requesting does not necessarily mean getting:
```suggestion
* sample resource metrics for the agent's node via the ResourceMonitor
```
